coordinator/Dockerfile

```diff
@@ -1,20 +1,38 @@
-FROM openjdk:22-ea-17-slim-bookworm
+# BUILDER image
+FROM eclipse-temurin:21-jdk-noble AS builder
```

Comment:
An image with JDK is needed here as we're using jar command to extract and recreate archive below, if any welcome to suggest a good way to use jar command without JDK